assets/js/util.js

```diff
@@ -242,8 +243,29 @@ function synchronizeTextareaHeights() {
     $('#translatedText').css('height', originalTextScrollHeight + 'px');
 }
 
-/*:: export {synchronizeTextareaHeights, modeEnabled, ajaxSend, ajaxComplete, filterLangList, onlyUnique,
-    SPACE_KEY_CODE, ENTER_KEY_CODE, HTTP_OK_CODE, HTTP_BAD_REQUEST_CODE, XHR_LOADING, XHR_DONE} */
+function ajaxCallForAllModes(methodType, request, endpoint, callbackSuccess, callbackError, requestName) {
```

Comment:
Call it either `successCallback` or `success`, `callbackSuccess` is unconventional.